torch/nn/functional.py

```diff
@@ -3776,10 +3781,12 @@ def upsample(input, size=None, scale_factor=None, mode="nearest", align_corners=
 
 def _is_integer(x) -> bool:
     r"""Type check the input number is an integer.
-    Will return True for int, SymInt and Tensors with integer elements.
+    Will return True for int, SymInt and Tensors / Numpy arrays with integer elements.
```

Comment:
Hm to clarify the docstring it is not actually an numpy array but rather a numpy integer is that correct?
```suggestion
    Will return True for int, SymInt, Numpy integers and Tensors with integer elements.
```

Reply:
Oops yes